services/director-v2/src/simcore_service_director_v2/models/comp_runs.py

```diff
@@ -18,6 +18,10 @@ class ProjectMetadataDict(TypedDict, total=False):
     parent_node_name: str
     parent_project_id: ProjectID
     parent_project_name: str
+    root_parent_project_id: ProjectID
+    root_parent_project_name: str
+    root_parent_node_id: NodeID
+    root_parent_node_name: str
```

Comment:
Question: Are these also supposed to be picked up by the `osparc` client from the environment and propagated through the api-server?

Reply:
no. I take care of this.